docs/getting-started/create-comit-app.md

```diff
@@ -4,6 +4,312 @@ title: Getting Started
+- [`comit-scripts`](https://www.npmjs.com/package/comit-scripts)
+
+While you can build your app in JavaScript, examples and the [SDK](https://github.com/comit-network/comit-js-sdk/) are written in TypeScript,
+allowing you to also build in TypeScript.
+
+To use either packages, you need to have [`yarn`](https://yarnpkg.com/) or [`npm`](https://www.npmjs.com/get-npm) installed.
+In this tutorial we will use `yarn` but it can be replaced by `npm` in any command below. 
+
+## TL;DR - Quick Start
+
+```
+yarn create comit-app my-app
+cd my-app/demos/btc_eth
+yarn install
+yarn run start-env
+```
+
+Once it says the `Environment is ready`, open a new terminal (keep the current terminal running):
+
+```
+cd my-app/demos/btc_eth
+yarn run swap
+```
+
+Congratulations, you did an atomic swap!
+
+## Long version
+
+### create-comit-app
+
+The `create-comit-app` allows you to create a template COMIT App which includes working examples.
+To create your template app in the `my-app` directory, just do:
+
+```
+yarn create comit-app my-app
+```
+```
+yarn create v1.21.1
+[1/4] 🔍  Resolving packages...
+[2/4] 🚚  Fetching packages...
+[3/4] 🔗  Linking dependencies...
+warning " > tscodeshift@0.4.1" has incorrect peer dependency "typescript@^2.3.2".
+warning "tscodeshift > ts-emitter@0.3.1" has incorrect peer dependency "typescript@^2.3.2".
+[4/4] 🔨  Building fresh packages...
+success Installed "create-comit-app@0.8.3" with binaries:
+      - create-comit-app
+First time execution, downloading create-comit-app 0.8.3...✓
+Your project `my-app` has been created!
+✨  Done in 7.09s.
+```
+
+Let's have a look at what we have here:
+
+```
+cd my-app
+tree # You can alternatively use `ls -R`
+```
+```
+.
+├── README.md
+├── demos
+│   ├── btc_eth
+│   │   ├── README.md
+│   │   ├── package.json
+│   │   ├── src
+│   │   │   └── index.ts
+│   │   ├── tsconfig.json
+│   │   ├── tslint.json
+│   │   └── yarn.lock
+│   └── erc20_btc
+│       ├── README.md
+│       ├── package.json
+│       ├── src
+│       │   └── index.ts
+│       ├── tsconfig.json
+│       ├── tslint.json
+│       └── yarn.lock
+├── examples
+│   ├── README.md
+│   └── btc_eth
+│       ├── README.md
+│       ├── package.json
+│       ├── src
+│       │   ├── lib.ts
+│       │   ├── maker.ts
+│       │   └── taker.ts
+│       ├── tsconfig.json
+│       ├── tslint.json
+│       └── yarn.lock
+├── package.json
+└── src
+    └── index.js
+
+9 directories, 24 files
+```
+
+- `.` is where you can start writing your app, see [write your app tuto](TODO) 
+- `./package.json` contains some useful dependencies for your app
+- `./examples` contains projects that are made to inspire developers such as yourself to build their app
+- `./demos` contains projects which are catered for workshops demonstration. They do not have a conventional structure but have verbose output.
+ 
+ Let's have a look at [comit-scripts](#comit-scripts) before [running an example](TODO) together.
```

Comment:
Same as above. Let's not lose track of these TODOs.

Reply:
Yep, thanks, done.